constants/modals.ts

```diff
@@ -7,13 +7,17 @@ export const DEPOSIT_MODAL = {
     name: 'open_options',
     number: 1
   },
+  OPEN_BUY_CRYPTO: {
+    name: 'open_buy_crypto',
+    number: 2,
+  },
   OPEN_FORM: {
     name: 'open_form',
-    number: 2,
+    number: 3,
```

Comment:
@SametSahin10 buy crypto and form are at the same level, after options either buy crypto can be opened or form, so both needs to be at number `2`